OpenRA.Mods.Common/Widgets/Logic/Editor/ActorEditLogic.cs

```diff
@@ -31,7 +31,7 @@ enum ActorIDStatus { Normal = 0, Duplicate = 1, Empty = 3 }
 		readonly BackgroundWidget actorEditPanel;
 		readonly LabelWidget typeLabel;
 		readonly TextFieldWidget actorIDField;
-		readonly LabelWidget actorIDErrorLabel;
+		LabelWidget actorIDErrorLabel;
```

Comment:
This can stay `readonly` as that only applies to initializing and not changing it's properties.

Reply:
Gotcha.  Corrected this and pushed.